update_ui.py

```diff
@@ -0,0 +1,71 @@
+from subprocess import Popen, PIPE
+from ui.utility import find_ui_update
+from ui.config_loader import load
+import os
+import logging
+from datetime import datetime
+import shutil
+import tarfile
+
+'''
+UI updater
+'''
+
+# setup the logs
```

Comment:
yep, but I thought we could use a differently named log. Thinking about it I think it's fine all to go in the same log as the programs won't be running at the same time.